canonical/lxd-ui
# fix: remove bulk instance migration feature due to server loading concerns

## Done

- From the discussion with lxd team, there are concerns about potentially overloading the server if bulk migration is performed on a large number of instances. For now, we will remove the feature and wait for the team to add background task scheduling capability so we can do bulk actions in the UI without issues.

Fixes [list issues/bugs if needed]

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - For the UI connected to a lxd cluster, check that the `Migrate` button is no longer visible when selecting multiple instances on the instance list page.
